### PR TITLE
fix: connect more user profile settings to the API

### PR DESCRIPTION
**Pull request by brandonroberts**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR:
- Connects `Display Local Time` and `Timezone` to the API for updates

## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
I am seeing weirdness when selecting a timezone. I can't select Mountain Time, it defaults to La Paz. 

Screen Shot 2023-02-01 at 1 39 31 PM


I chose mountain time, but it shows La Paz
Screen Shot 2023-02-01 at 1 39 37 PM

Should be UTC-7
Screen Shot 2023-02-01 at 1 40 06 PM



cc @OgDev-01

**Comment by brandonroberts** [comment]
@bdougie should be fixed now. I thought we could just use the offset, but its not unique so I switched it back to the string value

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx [hunk 1]
```diff
@@ -64,14 +68,14 @@ const ContributorProfileInfo = ({
...
+              {displayLocalTime && <span className="flex gap-2 items-center">
                 <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
-              </span>
+              </span>}
 
               <span className="flex gap-2 items-center">
                 <FiTwitter className="text-light-slate-9" />
                 <Link href="#" className="w-max hover:text-orange-500 ">
-                  coming soon...
+                  {twitterUsername || "coming soon..."}
```
Comment (OgDev-01) [inline components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:78]: Do we still need the coming soon here? IMHO, this should be off if a user twitter username is null and display when a it's available for the user 🍕

Comment (bdougie) [reply components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:78]: It should not show if it doesn't exist. Coming soon is not needed.


Also, TBH. I never meant for this to show on the user profile. It was meant for the settings side as a placeholder.

**Comment by bdougie** [comment]
This should still be UTC-8

Screen Shot 2023-02-01 at 1 52 52 PM

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx (hunk 1 above)
Comment (brandonroberts) [reply components/molecules/ContributorProfileInfo/contributor-profile-info.tsx]: Fixed

**Comment by brandonroberts** [comment]
> This should still be UTC-8, and not the "Pacific Standard Time"
> 
> Screen Shot 2023-02-01 at 1 52 52 PM

Fixed

**Review by bdougie — APPROVED** [review]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

**Review by OgDev-01 — APPROVED** [review]
LGTM 🍕🍕🍕

**Pull request merged**